client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { JsonForms } from '@jsonforms/react';
+import { materialRenderers } from '@jsonforms/material-renderers';
+import {
+  stringTester,
+  TextField,
+  selectTester,
+  Selector,
+  groupTester,
+  Group,
+  labelTester,
+  Label,
+  dateTester,
+  Date,
+  // arrayTester,
+  // Array,
+} from './components';
+
+// Temporarily hard-coded examples until we connect to database
+import patient from './jsonTemp/patient_1.json';
+import schema from './jsonTemp/schema.json';
+import uiSchema from './jsonTemp/ui-schema.json';
+import { JsonFormsRendererRegistryEntry } from '@jsonforms/core';
+
+type JsonData = {
+  [key: string]: string | number | boolean | null | unknown | JsonData;
+};
+
+interface JsonFormsComponentProps {
+  data: JsonData;
+  setData: (data: JsonData) => void;
+  renderers: JsonFormsRendererRegistryEntry[];
+}
+
+const FormComponent = ({
+  data,
+  setData,
+  renderers,
+}: JsonFormsComponentProps) => {
+  return (
+    <JsonForms
+      schema={schema}
+      uischema={uiSchema}
+      data={data}
+      renderers={renderers}
+      // cells={materialCells}
+      onChange={({ errors, data }) => {
+        setData(data);
+        console.log(errors);
```

Comment:
temporary? if not - perhaps a note to say where they've come from? 

Log conditionally? if there are no errors, then there's no need to log

```
if (errors && errors.length) console.warn('Errors: ', errors);
```